ak_survey_results.py

```diff
@@ -51,13 +47,14 @@ def __init__(self, settings):
         """
         Initialize settings.
         """
+        db_settings = get_secret('redshift-admin')
```

Comment:
This repository actually has tests which make use of a `test_settings.py` file, which provides credentials to a real database, but one that only contains test data, since the tests actually create and destroy data.

The way this is set up, the test database credentials will be ignored and the real ones will be used for testing. I think we either want to overhaul the test setup (perhaps using mocks) or pass the database credential secret name in to the `__init__` as an optional parameter, and use the local settings file when it is missing (which it will be, in the test code). But I'm also open to other solutions you might have!

Reply:
Thanks!  I updated the test code to use mock secrets, though testing will be complicated because it requires a local SQL database to use.